sdk/dotnet/Branch.cs

```diff
@@ -14,6 +14,28 @@ namespace Pulumi.Github
     /// 
     /// Additional constraints can be applied to ensure your branch is created from
     /// another branch or commit.
+    /// 
+    /// ## Example Usage
+    /// 
+    /// 
+    /// 
+    /// ```csharp
+    /// using Pulumi;
+    /// using Github = Pulumi.Github;
```

Comment:
Our casing of `GitHub` is wrong too... It looks like that applies to most new providers :(